cypress/fixtures/mockUsers.json

```diff
@@ -0,0 +1,6 @@
+[
+    { "username": "user1", "password": "password" },
+    { "username": "user2", "password": "password2" }
```

Comment:
I think we should include some mock data for the second user so that we can use it.

Reply:
Yes! was waiting for PR standup to discuss what other mock data can we add into this user as we progress.